plugin/src/main/resources/play/soap/plugin/sei.vm

```diff
@@ -1,5 +1,7 @@
 ##
-## Copyright © 2015 Lightbend Inc. <https://www.lightbend.com>
+## Copyright (C)
+## From 2022 The Play Framework Contributors <https://github.com/playframework>
+## 2011-2021 Lightbend Inc. <https://www.lightbend.com>
```

Comment:
I put it in three lines in this vm file because otherwise the file looks a bit weird since it almost just has short lines.